code/modules/clothing/suits/wintercoats.dm

```diff
@@ -629,6 +632,9 @@
 		/obj/item/storage/bag/ore,
 		/obj/item/t_scanner/adv_mining_scanner,
 		/obj/item/tank/internals,
+		/obj/item/shovel,
+		/obj/item/shovel/serrated,
```

Comment:
```suggestion
		/obj/item/shovel,
```
The compile-time list of allowed suit-slotted items is typecached on runtime iirc.